src/Parser/DemonHunter/Vengeance/Modules/Spells/SigilOfFlame.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SPELLS from 'common/SPELLS/index';
+import SpellIcon from 'common/SpellIcon';
+
+import { formatPercentage, formatThousands, formatDuration } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+/*Purpose of this module is to track the sigil of flame debuff and see when 2 of them overlap for a damage increase.
+* This is important for damage and also squeezing extra fire damage during Fiery Brand because of the
+* Fiery Demise talent that increases your fire damage during that CD. Also due to trait charred blades that heals
+* us for 15% of the fire damage we do it also increases our self healing.*/
+class SigilOfFlame extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    enemies: Enemies,
+  };
+
+  successfulStack = 0;
+  lastApplicationTimestamp = 0;
+  currentApplicationTimestamp = 0;
+
+  on_byPlayer_applydebuff(event){
+    const spellId = event.ability.guid;
+    let timeStampDifference = null;
+
+    if(spellId !== SPELLS.SIGIL_OF_FLAME_DEBUFF.id) {
+      return;
+    }
+    if(this.lastApplicationTimestamp === 0){
+      this.lastApplicationTimestamp = event.timestamp;
+      return;
+    }else{
```

Comment:
`else` can be omitted since you already return in the previous block

Reply:
Removed